packages/styleguide/src/components/TeaserCarousel/Tile.js

```diff
@@ -135,7 +135,7 @@ const Tile = ({
       borderWidth: outline && 1,
       borderStyle: outline && 'solid',
       cursor: onClick ? 'pointer' : 'default',
-      padding: bigger ? '0 0 20px 0' : '30px 15px',
+      padding: bigger & !outline ? '0 0 20px 0' : '30px 15px',
```

Comment:
In my mind we would have done the padding only around the text and keep the images as big as possible: 
<img width="1079" alt="Screenshot 2022-02-10 at 13 03 05" src="https://user-images.githubusercontent.com/410211/153405156-a3239926-9127-436c-89e2-3f08a6ab196e.png">

Could you add an example in the styleguide with the proposed content by Sven and check how a while row looks? Would also be good to have this combination documented in the styleguide.